lib/OBSWebSocket.js

```diff
@@ -37,20 +35,20 @@ class OBSWebSocket extends Socket {
       const messageId = args['message-id'] = generateMessageId(); // eslint-disable-line no-multi-assign
 
       // Submit the request to the websocket.
-      log.debug('[Core:send]', messageId, requestType, args);
+      debug('[send] %s %s %O', messageId, requestType, args);
       this._socket.send(JSON.stringify(args));
```

Comment:
What I had meant though is that if there's no socket connection sending any request produces an error here, which apparently isn't used as a reject. That's a separate issue from wrapping the `.connect()` function.

```
obs.getCurrentScene().then(data => { console.log('success'); }).catch(err => { console.log(err); });
> WebSocket is already in CLOSING or CLOSED state.
Promise {[[PromiseStatus]]: "pending", [[PromiseValue]]: undefined}
```